### PR TITLE
Create stealmoji Command.

### PR DESCRIPTION
**Pull request by vypr-ysl**

### CODE REVIEW

**Review by l1ving — APPROVED** [review]

**Comment by humboldt123** [comment]
Please check the max amount of emojis per guild and display a warning if you cannot add an emoji due to its restriction.

**Comment by l1ving** [comment]
https://github.com/kami-blue/bot-kt/blob/eafaac4d8e5250c83d18954f5f46853b840de9fd/src/main/kotlin/util.kt#L197

Here's my code for getting max guild emoji

**Comment by humboldt123** [comment]
For DiscordJS you can simply check the [premium tier](https://discord.js.org/?source=post_page---------------------------#/docs/main/stable/typedef/PremiumTier) of the server and do it based off of that.
| Tier   |  Emoji Slots |
|----------|:-------------:|
| 0 | 50|
| 1 | 100 |
| 2 | 150 |
| 3 | 250 |

**Comment by l1ving** [comment]
That's what the code I sent does <:woeis:759835843467673644>

**Comment by humboldt123** [comment]
woeis

**Comment by vypr-ysl** [comment]
emojiCount & Permission Checks have been added.

File: commands/stealmoji.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    if(message.guild.emojis.cache.size > maxEmoji) return message.channel.send('Guild has reached the maximum number emoji slots.')
+
+    if (!message.guild.members.cache.get(client.user.id).hasPermission("MANAGE_EMOJIS")) {
+        return message.channel.send(`\`❌\` Missing Permissions: \`MANAGE_EMOJIS\``)
+    }
+
+    if(!message.guild.members.cache.get(message.author.id).hasPermission("MANAGE_EMOJIS")) {
+        return message.channel.send(`\`❌\` Missing Permissions: \`MANAGE_EMOJIS\``)
+    }
```
Comment (l1ving) [inline commands/stealmoji.js:24]: make the error messages more clear, so you know who's missing perms

**Review by l1ving — CHANGES_REQUESTED** [review]

File: commands/stealmoji.js (hunk 1 above)
Comment (vypr-ysl) [reply commands/stealmoji.js]: Added.

**Review by l1ving — APPROVED** [review]

File: commands/stealmoji.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+const emojiRegex = /^<:[\w\d]*:\d{18}>$/g
+
+/**
+ * @author vypr-ysl
+ * @command stealmoji
+ **/
+
+module.exports.run = async (client, message, args) => {
+
+    const guildTier = message.guild.premiumTier
```
Comment (humboldt123) [inline commands/stealmoji.js:10]: Please use a switch statement.

**Comment by humboldt123** [comment]
Alternatively use this:
```js
maxEmoji 50 + (((guildTier == 3)?4:guildTier)*50);
```

**Comment by vypr-ysl** [comment]
Switch Statements have been added.

**Comment by humboldt123** [comment]
Thank you.

**Comment by vypr-ysl** [comment]
No problem.

**Pull request merged**